static/script.js

```diff
@@ -398,11 +402,14 @@ function picksubtree(n) {
 			tree: editor.getValue()
 		},
 		success: function(response) {
-			var res = response.split('\t', 2);
+			var res = response.split('\t', 3);
```

Comment:
descriptive variable names for the individual fields would be better (or make the response an object with named fields)